src/Microsoft.AspNetCore.Razor.Language/DefaultImportDiscoverer.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    internal class DefaultImportDiscoverer : RazorProjectEngineFeatureBase, IRazorImportDiscoverer
```

Comment:
I think it's good to expose this as a base class or something since it's what folks are used to MVC doing.